### PR TITLE
Bloodpack resprite and visual upgrade

### PR DESCRIPTION
**Pull request by Blundir**
## About The Pull Request

Resprites bloodpack, adding actual reagent filling bar (like on beakers, hyposprays, etc), adds blood type icon on every blood type bloodpack.
dreamseeker_KDq4AICpBw

dreamseeker_UxI8P3rtRo

## Why It's Good For The Game

Part of the reject CM cycle, here is how it looked before:
dreammaker_CZam5QVZuL
Literally 3 icons, no custom reagent visual support.

New sprites look better, more fitting, provides more functionality.

## Changelog

:cl:
add: added reagent filling overlay, blood type overlay for bloodpack
imageadd: added new icons for bloodpacks 
/:cl:

### CODE REVIEW

**Comment by Lumipharon** [comment]
old ones look sharp enough to cut

**Comment by lbnesquik** [comment]
I prefer the old ones, honestly.

**Comment by Al-1ce** [comment]
The bloodpack meta begins.

File: code/game/objects/items/reagent_containers/blood_pack.dm [hunk 1]
```diff
@@ -13,22 +13,39 @@
...
+			if(80 to 90)
+				filling.icon_state = "[initial(icon_state)]80"
+			if(91 to INFINITY)
+				filling.icon_state = "[initial(icon_state)]100"
+
+		filling.color = mix_color_from_reagents(reagents.reagent_list)
+		overlays += filling
+
+	if(blood_type)
+		add_overlay(image('icons/obj/items/bloodpack.dmi', "[blood_type]"))
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/blood_pack.dm:48]: incorrect update overlays usage
The way it works is you call update_icon and then update_overlays returns a list of overlays to add

**Review by TiviPlus — CHANGES_REQUESTED** [review]

**Review by Kuro020 — APPROVED** [review]

**Pull request closed without merging**